--- requirements	2015-04-24 19:51:06.000000000 +0000
+++ requirements1	2015-05-08 12:49:28.784822000 +0000
@@ -11,6 +11,16 @@
 
 private
 
+pattern New_ConfigGuess \
+ private ; \
+ macro_append constituents " New_ConfigGuess" ;\
+ action New_ConfigGuess "cd ../src ; tar -xvf lzo-2.0.3.tar.gz ; rm lzo-2.0.3/autoconf/config.guess ; cp /usr/share/automake-1.14/config.guess lzo-2.0.3/autoconf ; rm lzo-2.0.3.tar.gz ; tar -czv lzo-2.0.3 -f lzo-2.0.3.tar.gz ; rm -r lzo-2.0.3 ; cd ../cmt " ; \
+ end_private
+
+apply_pattern New_ConfigGuess
+
+
+
 apply_pattern make_pkgbuild \
  name=lzo \
  file=pkgbuild_lzo.py
