--- gccxml_plugin.h	2014-11-12 19:08:27.279057553 +0200
+++ gccxml_plugin1.h	2014-11-12 19:21:37.244075694 +0200
@@ -39,7 +39,7 @@
 #include "gcc-plugin.h"
 
 #define GCC_XML_C_VERSION "GCC-XML Plugin Version: " GCCXML_PLUGIN_VERSION
-
+//int plugin_is_GPL_compatible;
 /* These were hacked in to GCC's c-common/c.opt file, so don't exist for
  * the plugin. We populate them in the callback routine,
  * `collect_callback_args`.   */
