--- AthenaServices_entries.cxx	2015-02-03 11:31:27.560911236 +0200
+++ AthenaServices_entries.cxx1	2015-02-03 11:32:15.390627747 +0200
@@ -30,7 +30,7 @@
 #include "../MetaDataSvc.h"
 #include "../OutputStreamSequencerSvc.h"
 #include "../AthenaSharedMemoryTool.h"
-#include "../AthenaYamplTool.h"
+//#include "../AthenaYamplTool.h"
 #ifdef ATHENAHIVE
 #include "../AthenaHiveEventLoopMgr.h"
 #endif
@@ -69,7 +69,7 @@
     DECLARE_TOOL( AthenaOutputStreamTool )
     DECLARE_TOOL( ThinningOutputTool )
     DECLARE_TOOL( AthenaSharedMemoryTool )
-    DECLARE_TOOL( AthenaYamplTool )
+    //DECLARE_TOOL( AthenaYamplTool )
 
 }
 DECLARE_ALGORITHM_FACTORY( AthenaOutputStream )
@@ -105,6 +105,6 @@
 DECLARE_TOOL_FACTORY( AthenaOutputStreamTool )
 DECLARE_TOOL_FACTORY( ThinningOutputTool )
 DECLARE_TOOL_FACTORY( AthenaSharedMemoryTool )
-DECLARE_TOOL_FACTORY( AthenaYamplTool )
+//DECLARE_TOOL_FACTORY( AthenaYamplTool )
 
 
