--- requirements	2015-01-23 13:05:56.888928135 +0200
+++ requirements1	2015-02-25 10:16:44.903414130 +0200
@@ -93,6 +93,7 @@
 
 pattern lcgdict \
   private ; \
+  macro_remove AtlasCxxPolicy_pp_cppflags " -fPIC" ; \
   use Reflex         v*               LCG_Interfaces ; \
   macro gcc_user_flags "" target-darwin&target-x86_64 " -m64" ; \
   macro dict_dir "$(bin)dict" ; \
@@ -164,8 +165,8 @@
 #HACK, should be fixed in GaudiPolicy. Get rid of icc command line options
 macro_remove cppflags "" target-icc "-wd383 -wd981 -wd1418 -wd1419 -we1011 -we117 -wd82 -wd522 -wd444 -wd304 -Wno-deprecated"
 
-macro_prepend cppflags " -Wno-literal-suffix -I$(gcc_home)/lib/gcc/x86_64-unknown-linux-gnu/$(gcc_native_version)/plugin/include -I$(gcc_home)/lib/gcc/x86_64-unknown-linux-gnu/$(gcc_native_version) -I$(gmp_home)/include -DGCCXML_PLUGIN_VERSION=\$(q)1.0sss2\$(q) -DGCCXML_PLUGIN_VERSION_FULL=\$(q)1.0.2sss2\$(q) "
-macro_prepend cflags " -I$(gcc_home)/lib/gcc/x86_64-unknown-linux-gnu/$(gcc_native_version)/plugin/include  -I$(gcc_home)/lib/gcc/x86_64-unknown-linux-gnu/$(gcc_native_version)  -DGCCXML_PLUGIN_VERSION_FULL=\$(q)1.0.2sss2\$(q) -I$(gmp_home)/include "
+macro_prepend cppflags " -Wno-literal-suffix -I/usr/lib/gcc/armv7hl-redhat-linux-gnueabi/4.9.2/plugin/include -I/usr/lib/gcc/armv7hl-redhat-linux-gnueabi/4.9.2 -I$(gmp_home)/include -DGCCXML_PLUGIN_VERSION=\$(q)1.0sss2\$(q) -DGCCXML_PLUGIN_VERSION_FULL=\$(q)1.0.2sss2\$(q) "
+macro_prepend cflags " -I/usr/lib/gcc/armv7hl-redhat-linux-gnueabi/4.9.2/plugin/include  -I/usr/lib/gcc/armv7hl-redhat-linux-gnueabi/4.9.2  -DGCCXML_PLUGIN_VERSION_FULL=\$(q)1.0.2sss2\$(q) -I$(gmp_home)/include "
 
 # Remove -m32 from compilation lines.  Need to build to match the compiler used.
 macro_remove cflags " -m32 "
