--- requirements	2015-04-02 13:19:13.570628084 +0200
+++ requirements1	2015-04-02 13:18:52.685757348 +0200
@@ -16,7 +16,7 @@
 ## build libraries
 apply_pattern dual_use_library files="../Root/*.cxx ../src/*.cxx ../src/components/*.cxx"
 macro FileStagerLib_dependencies  "filestager_root_dictionary"
-action  filestager_root_dictionary  "( make -f Makefile.Standalone stagercint clean )"
+action  filestager_root_dictionary  "( make -f Makefile stagercint clean )"
 
 ### Create a named installed library
 #library FileStagerLib "../Root/*.cxx"
