--- requirements	2015-04-13 22:20:01.230869861 +0200
+++ requirements1	2015-04-15 16:13:37.229107315 +0200
@@ -93,6 +93,10 @@
 tag gcc483      host-gcc48
 tag host-gcc48  host-gcc4
 
+tag gcc491	host-gcc49
+tag gcc492	host-gcc49
+tag host-gcc49	host-gcc49
+
 # Windows Visual Studio cl compiler
 tag VisualC     host-visualc
 tag cl          host-cl
@@ -107,6 +111,7 @@
 tag WIN32       host-win7
 tag XP          host-winxp
 tag_exclude     host-winxp host-win7
+tag fc21	host-fc21
 # slc4 series (latest release 4.7)
 tag slc40       host-slc4
 tag slc41       host-slc4
@@ -278,6 +283,9 @@
 tag x86_64-win7-vc9-opt      target-x86_64  target-win7    target-vc9      target-opt
 tag x86_64-win7-vc9-dbg      target-x86_64  target-win7    target-vc9      target-dbg
 
+tag armv7l-fc21-gcc49-opt    target-armv7l  target-fc21	   target-gcc49    target-opt
+tag armv7l-fc21-gcc49-dbg    target-armv7l  target-fc21    target-gcc49    target-dbg
+
 # Common testing tags (i.e. not foreseen to be used for release)
 tag i686-slc5-icc11-opt      target-i686   target-slc5   target-icc11  target-opt
 tag i686-slc5-icc11-dbg      target-i686   target-slc5   target-icc11  target-dbg
@@ -348,6 +356,7 @@
 tag target-darwin target-unix
 tag target-winxp  target-win
 tag target-win7   target-win
+tag target-fc21   target-linux
 
 tag target-gcc34    target-gcc3
 tag target-gcc40    target-gcc4
@@ -397,7 +406,8 @@
       target-i386                   "i386"   \
       target-slc4&target-x86_64     "amd64"  \
       target-slc4&target-i686       "ia32"   \
-      target-mac105                 "ia32"
+      target-mac105                 "ia32"   \
+      target-armv7l		    "armv7l" 
 
 macro LCG_os                        "" \
       target-slc6                   "slc6"   \
@@ -408,7 +418,8 @@
       target-mac106                 "mac106" \
       target-mac105                 "osx105" \
       target-win7                   "win7"   \
-      target-winxp                  "winxp"
+      target-winxp                  "winxp"  \
+      target-fc21		    "fc21"  
 
 macro LCG_compiler                  "" \
       target-gcc34                  "gcc34"   \
@@ -476,8 +487,9 @@
          "slc4_amd64_gcc43"       "x86_64-slc4-gcc43-opt" \
          "slc4_amd64_gcc43_dbg"   "x86_64-slc4-gcc43-dbg" \
          "osx105_ia32_gcc401"     "i386-mac105-gcc40-opt" \
-         "osx105_ia32_gcc401_dbg" "i386-mac105-gcc40-dbg"
-
+         "osx105_ia32_gcc401_dbg" "i386-mac105-gcc40-dbg" \
+	 "armv7l-fc21-gcc49"	  "armv7l-fc21-gcc49-opt" \
+         "armv7l-fc21-gcc49_dbg"      "armv7l-fc21-gcc49-dbg" \
 # =====  Automatic setup of CMTCONFIG  =====
 
 macro host_compiler                     "UnknownCompiler" \
@@ -494,6 +506,7 @@
       host-gcc46                        "gcc46" \
       host-gcc47                        "gcc47" \
       host-gcc48                        "gcc48" \
+      host-gcc49			"gcc49" \
       host-gcc                          "unknownGcc"  \
       host-clang30                      "clang30" \
       host-clang31                      "clang31" \
@@ -512,7 +525,8 @@
       host-x86_64                       "x86_64" \
       host-i686                         "i686" \
       host-i386                         "i386" \
-      host-ppc                          "ppc" 
+      host-ppc                          "ppc"  \
+      host-armv7l			"armv7l"
 
 macro host_os                           "UnknownOS" \
       host-slc4                         "slc4" \
@@ -521,7 +535,8 @@
       host-debian                       "debian" \
       host-linux                        "linux" \
       host-darwin&host-gcc40            "osx104" \
-      host-darwin                       "darwin"
+      host-darwin                       "darwin" \
+      host-fc21				"fc21"
 
 set   host_cmtconfig                    "$(host_arch)-$(host_os)-$(host_compiler)-opt" \
       host-slc4                         "$(host_os)_$(host_arch)_$(host_compiler)" \
