--- requirements	2015-04-29 13:11:28.734822000 +0000
+++ requirements1	2015-04-30 11:24:29.234822000 +0000
@@ -17,7 +17,8 @@
       target-gcc46   "4.6.3"	\
       target-gcc47   "4.7.2"	\
       target-gcc48   "4.8.1"	\
-      target-gcc49   "4.9.0"	\	  
+      target-gcc49   "4.9.0"	\
+      target-fc21&target-aarch64   "4.9.2"	\  
       target-clang32 "4.6.3"    \
       target-clang33 "4.8.1"    \
       target-icc13   "4.8.1"
