--- requirements	2015-02-20 08:52:21.000000000 +0000
+++ requirements1	2015-05-05 10:52:49.744822000 +0000
@@ -38,7 +38,7 @@
 find @INSTALL_PREFIX@/sw/lcg/external/Python/${PYTHON_VERSION}/${lcg_bin_tag}/lib        \
  '(' -name '*.so' -or -name '*.so.*' ')' -maxdepth 1 -exec                                  \
  ln -snf '{}' @INSTALL_PREFIX@/$(dirname)/external/${CMTCONFIG}/lib/ \;  ;       \
-find @INSTALL_PREFIX@/sw/lcg/external/uuid/${UUID_VERSION}/${lcg_bin_tag}/lib        \
+find /home/seuster/LCGStack/lcgcmake-install/uuid/1.42/aarch64-ubuntu14.04-gcc49-opt/lib        \
  '(' -name '*.so' -or -name '*.so.*' ')' -maxdepth 1 -exec                                  \
  ln -snf '{}' @INSTALL_PREFIX@/$(dirname)/external/${CMTCONFIG}/lib/ \;  ;       \
 find @INSTALL_PREFIX@/sw/lcg/external/zlib/${ZLIB_VERSION}/${lcg_bin_tag}/lib        \
