--- requirements	2015-05-27 17:59:08.984822000 +0000
+++ requirements1	2015-05-27 17:58:59.974822000 +0000
@@ -5,7 +5,7 @@
 use AtlasPolicy         AtlasPolicy-*
 use PyCmt               PyCmt-*         Tools -no_auto_imports
 
-#macro_append AtlasShowerDeconstruction_linkopts "-lDeconstruction"
+macro_append AtlasShowerDeconstruction_linkopts "-lDeconstruction"
 include_path none
 apply_pattern include_path extras=$(CMTINSTALLAREA)/$(CMTCONFIG)/include
 
