--- SealDebug.cxx	2014-11-11 18:07:36.923953085 +0200
+++ SealDebug1.cxx	2014-11-14 12:24:13.177588220 +0200
@@ -441,7 +441,7 @@
 }
 
 
-#if __GNUC__ > 3 || (__GNUC__ == 3 && __GNUC_MINOR__ >= 4) // FIXME: Check
+//#if __GNUC__ > 3 || (__GNUC__ == 3 && __GNUC_MINOR__ >= 4) // FIXME: Check
 extern "C" {
     typedef unsigned _Unwind_Ptr __attribute__((__mode__(__pointer__)));
     struct _Unwind_Context;
@@ -469,7 +469,7 @@
     walking only through frames with DWARF2 exception data, so it may
     not be able to go through all C libraries.  It does seem to be
     able to signal frames on a number of platforms however.  */
-_Unwind_Reason_Code
+/*_Unwind_Reason_Code
 unwindWalkStack (_Unwind_Context *ctx, void *data)
 {
     IOFD		fd = *(IOFD *) data;
@@ -532,7 +532,7 @@
     return _URC_NO_REASON;
 }
 #endif // GCC 3.4+
-
+*/
 //<<<<<< PUBLIC FUNCTION DEFINITIONS                                    >>>>>>
 
 // Change the path of the binary used for symbolization.
