--- requirements	2015-01-28 18:30:40.283136147 +0200
+++ requirements1	2015-01-28 18:34:42.846681980 +0200
@@ -48,7 +48,7 @@
 macro_append DOXYGEN_FILE_PATTERNS    " *.icc"
 
 application genCLIDDB ../utils/genCLIDDB.cxx 
-macro_append genCLIDDBlinkopts " $(Boost_linkopts_program_options) "
+macro_append genCLIDDBlinkopts " $(Boost_linkopts_program_options) -lAthenaKernel -lGaudiKernel -L$(GaudiDir)/InstallArea/$(CMTCONFIG)/lib"
 macro_append genCLIDDB_dependencies " AthenaKernel"
 
 #
