--- OldCLHEPStreamers.cxx	2015-02-03 10:05:20.525914870 +0200
+++ OldCLHEPStreamers.cxx.corrected	2015-02-03 10:06:06.100660765 +0200
@@ -19,7 +19,7 @@
 //      std::cout << "  matrix ncol="<< old_matx.ncol  << std::endl;
 //      std::cout << "  matrix m-size="<< old_matx.m.size()  << std::endl;
      new_matx.m = old_matx.m;    //   ----   FIXME
-     new_matx.size_ = old_matx.size;
+     new_matx.size = old_matx.size;
      // intentional swap!   somehow necessary for reading root3 data 
      new_matx.nrow = old_matx.ncol;
      new_matx.ncol = old_matx.nrow;
