--- requirements	2015-02-03 10:42:55.258180167 +0200
+++ requirements1	2015-02-03 10:43:02.733136486 +0200
@@ -20,7 +20,7 @@
 private
 use GaudiInterface       GaudiInterface-*       External
 use AthenaKernel         AthenaKernel-*         Control
-use AtlasDSFMT           AtlasDSFMT-*           External
+#use AtlasDSFMT           AtlasDSFMT-*           External
 use AthenaBaseComps      AthenaBaseComps-*      Control
 use AtlasROOT            AtlasROOT-*            External
 use AtlasBoost           AtlasBoost-*           External
