--- requirements	2015-01-20 13:36:28.297673116 +0200
+++ requirements1	2015-02-09 16:22:23.471191076 +0200
@@ -25,7 +25,7 @@
 macro DOXYGEN_IMAGE_PATH "../doc/images"
 macro DOXYGEN_INLINE_INHERITED_MEMB "YES"
 use AtlasReflex   AtlasReflex-*   External -no_auto_imports
-apply_pattern lcgdict dict=TrkParameters selectionfile=selection.xml headerfiles="../TrkParameters/TrkParametersDict.h"
+#apply_pattern lcgdict dict=TrkParameters selectionfile=selection.xml headerfiles="../TrkParameters/TrkParametersDict.h"
 
 
  
