--- requirements	2015-04-24 19:41:31.000000000 +0000
+++ requirements1	2015-05-04 11:48:06.204822000 +0000
@@ -151,12 +151,14 @@
       slc5          "/usr" \
       slc6          "/usr" \
       Darwin        "/usr" \
-      x86_64&gcc    "${ATLAS_EXTERNAL}/fortran/$(AtlasFortranPolicy_native_version)/gfortran64"
+      x86_64&gcc    "${ATLAS_EXTERNAL}/fortran/$(AtlasFortranPolicy_native_version)/gfortran64" \
+      ubuntu	    "/usr/lib/gcc/aarch64-linux-gnu/4.9" 
 
 macro GFO_LIBS      "lib" \
       slc4&64       "lib64" \
       slc5&64       "lib64" \
-      slc6&64       "lib64" 
+      slc6&64       "lib64" \
+      ubuntu 	    "$(GFO_BASE)"
       
 macro GFO           "" \
    slc4&gcc-3.2.3   "$(GFO_BASE)/irun" \
@@ -168,7 +170,8 @@
    Darwin           "$(GFO_BASE)/local/gfortran" \
    CERN             "$(GFO_BASE)/irun" \
    BNL              "$(GFO_BASE)/irun" \
-   STANDALONE       "$(GFO_BASE)/irun"
+   STANDALONE       "$(GFO_BASE)/irun" \
+   ubuntu	    "/usr/bin/gfortran" 
 
 # These two macros are identical to the asNeeded_linkopt and notAsNeeded_linkopt macros defined in AtlasPolicy
 # but are needed here because this package doesn't have a dependency against AtlasPolicy
@@ -203,14 +206,15 @@
      HAS_GFO&slc6   " $(GFO)/$(GFO_LIBS)/libgfortran.so.1" \
      HAS_GFO&Darwin " $(GFO)/$(GFO_LIBS)/libgfortran.dylib" \
      HAS_GFO        " $(GFO)/$(GFO_LIBS)/libgfortranbegin.a $(GFO)/$(GFO_LIBS)/libgfortran.a" \
-     HAS_G95        " $(G95)/lib/gcc-lib/i686-pc-linux-gnu/4.0.1/libf95.a"
-
+     HAS_G95        " $(G95)/lib/gcc-lib/i686-pc-linux-gnu/4.0.1/libf95.a" \
+     HAS_GFO&ubuntu " /usr/lib/gcc/aarch64-linux-gnu/4.9/libgfortran.so"
 
 macro HAS_GFO_export "" \
       HAS_GFO&slc4   "" \
       HAS_GFO&slc5   "" \
       HAS_GFO&slc6   "" \
-      HAS_GFO "$(GFO)/bin $(GFO)/include $(GFO)/$(GFO_LIBS)  $(GFO)/libexec "
+      HAS_GFO "$(GFO)/bin $(GFO)/include $(GFO)/$(GFO_LIBS)  $(GFO)/libexec " \
+      HAS_GFO&ubuntu   "" 
 
 #
 # Settings only appropriate when the full compiler is available
