--- FPEAudit_linux.icc	2015-02-03 13:45:19.052954623 +0200
+++ FPEAudit_linux.icc1	2015-02-03 13:47:28.472180047 +0200
@@ -315,19 +315,19 @@
     
 #endif
 
-    void mask_fpe(void)
-    {
-      mask_x87();
-      if (cpu_has_sse2())
-        mask_sse2();
-    }
-    
-    void unmask_fpe(void)
-    {
-      unmask_x87();
-      if (cpu_has_sse2())
-        unmask_sse2();
-    }
+   // void mask_fpe(void)
+   // {
+  //   mask_x87();
+  //    if (cpu_has_sse2())
+  //     mask_sse2();
+  // }
+   // 
+  // void unmask_fpe(void)
+   // {
+   //   unmask_x87();
+   //   if (cpu_has_sse2())
+   //     unmask_sse2();
+   //}
     
     // printing of stacktrace including inlined functions. needs debug symbols
     // uses libbdf and libiberty from gdb, which currently seemed to have a
