--- RootCollectionQuery.cpp	2015-05-02 17:26:35.334822000 +0000
+++ RootCollectionQuery.cpp~	2015-05-02 17:27:26.194822000 +0000
@@ -201,7 +201,7 @@
         // increase number of operators ROOT can handle for complex queries
         coral::MessageStream log("APR::RootCollectionQuery");
         log << coral::Info << "Setting TFormula::SetMaxima(" << maxoper << ")" <<  coral::MessageStream::endmsg;
-        TFormula::SetMaxima( maxoper );
+        //TFormula::SetMaxima( maxoper );
      }
      APRTreeFormula formula(m_whereClause, m_tree);
      if( !formula.GetNdim() )  {
