--- get_files_header	2015-01-20 11:53:34.870049162 +0200
+++ get_files_header1	2015-01-28 17:23:08.922333069 +0200
@@ -26,7 +26,7 @@
 		     grep -v '=' | \
 		     sed -e 's#,##' | \
 		     sort -u` ; \
-	  get_files $(copymethod) -jo $${jonames} -scripts $${scriptnames} -data "$${datanames}" -xmls $${xmlnames}
+	  get_files=' $(copymethod) -jo $${jonames} -scripts $${scriptnames} -data "$${datanames}" -xmls $${xmlnames}'
 
 ${CONSTITUENT}clean ::
 	@if test -d ${install_dir}; then \
