--- requirements	2015-01-30 09:51:25.225781046 +0200
+++ requirements1	2015-01-30 09:48:40.506742000 +0200
@@ -410,7 +410,7 @@
    private ; \
      use GaudiKernel v* -no_auto_imports ; \
      macro dummy_for_library "<library>" ; \
-     macro <package>_genconfig_cmd "generate_configurables" no_genconf "do_null_genconfig" ; \
+     macro <package>_genconfig_cmd "generate_configurables" do_genconf "do_null_genconfig" ; \
      apply_pattern $(<package>_genconfig_cmd) library=<package> ; \
      apply_pattern generate_componentslist library=<package> group=<group> ; \
    end_private ; \
