--- requirements	2015-05-05 18:12:23.754822000 +0000
+++ requirements1	2015-05-05 18:06:04.074822000 +0000
@@ -49,7 +49,7 @@
 private
 
 macro_append TrigConfStorage_linkopts " $(Boost_linkopts_regex) \
--lboost_unit_test_framework-$(Boost_compiler_version)-mt-$(Boost_debug)$(Boost_file_version) "
+-lboost_unit_test_framework-$(Boost_compiler_version)-mt-$(Boost_debug)$(Boost_file_version) $(Boost_linkopts_system) "
 
 #============================================================================
 # Build applications
