--- CLHEP_Random.h	2015-03-13 16:27:37.365443622 +0200
+++ CLHEP_Random1.h	2015-03-13 16:27:27.940498076 +0200
@@ -3,8 +3,8 @@
 //----Functions missing in the implementation and required by the dictionary
 //    [ Bug #1806 ] Missing method implementations
 
-inline void CLHEP::RandPoissonT::shootArray(class CLHEP::HepRandomEngine *,int,long *,double) {}
-inline void CLHEP::RandPoissonT::fireArray(int,long *) {}
+//inline void CLHEP::RandPoissonT::shootArray(class CLHEP::HepRandomEngine *,int,long *,double) {}
+//inline void CLHEP::RandPoissonT::fireArray(int,long *) {}
 inline void CLHEP::RandPoissonQ::shootArray(class CLHEP::HepRandomEngine *,int,long *,double){}
 inline void CLHEP::RandPoissonQ::fireArray(int,long *) {}
 //// DRQ: The following function was removed in CLHEP 1.9.2.1
