--- IOVRegistrationSvc.cxx	2014-11-17 14:52:25.902606766 +0200
+++ IOVRegistrationSvc1.cxx	2014-11-17 14:54:42.146666486 +0200
@@ -981,10 +981,10 @@
 	    if (m_payloadTable)
 	      log << MSG::INFO << "Creating separate payload table for "
 		  << local_folder << endreq;
-	    cool::FolderSpecification folderSpec(
-		    version,payloadSpec,m_payloadTable);
-	    folder = db->createFolder(local_folder,folderSpec,
-                  mergedNames,true); 
+	    //cool::FolderSpecification folderSpec(
+	     //	    version,payloadSpec,m_payloadTable);
+	    //folder = db->createFolder(local_folder,folderSpec,
+            //      mergedNames,true); 
 	    log << MSG::DEBUG <<"Creation of CondDBFolder " << 
                    local_folder << " done" << endreq;
 
