--- requirements	2015-02-09 14:36:24.219071483 +0200
+++ requirements1	2015-02-09 15:30:22.839755096 +0200
@@ -18,7 +18,6 @@
 
 private 
 use CxxUtils            CxxUtils-*                  Control
-
 public
 
 library TrkSurfaces *.cxx
@@ -28,17 +27,15 @@
 # apply_tag  use_trkdetdescr_floatprecision 
 
 macro_append TrkDetDescrFloatPrecision " -DTRKDETDESCR_USEFLOATPRECISON "
- 
 macro_append  use_pp_cppflags                "" \
               use_trkdetdescr_floatprecision $(TrkDetDescrFloatPrecision)
 
 private
 
-
 #macro cppdebugflags '$(cppdebugflags_s)'
 #macro_remove componentshr_linkopts "-Wl,-s"
 
 macro DOXYGEN_IMAGE_PATH "../doc/images"
 macro DOXYGEN_INLINE_INHERITED_MEMB "YES"
-use AtlasReflex   AtlasReflex-*   External -no_auto_imports
-apply_pattern lcgdict dict=TrkSurfaces selectionfile=selection.xml headerfiles="../TrkSurfaces/TrkSurfacesDict.h"
+#use AtlasReflex   AtlasReflex-*   External -no_auto_imports
+#apply_pattern lcgdict dict=TrkSurfaces selectionfile=selection.xml headerfiles="../TrkSurfaces/TrkSurfacesDict.h"
