--- TConverterRegistry.cxx	2014-11-13 15:32:29.207650403 +0200
+++ TConverterRegistry1.cxx	2014-11-13 15:32:55.877482242 +0200
@@ -48,7 +48,7 @@
                                        bool takeown)
 {
   // Make sure the branch element is properly initialized.
-  TConvertingBranchElement_init();
+  //TConvertingBranchElement_init();
 
   // Look up the checksum->converter map.
   // (This will make an empty one if this is the first time we've
