--- requirements	2015-04-27 18:33:03.805090467 +0200
+++ requirements1	2015-04-27 18:34:29.059558937 +0200
@@ -53,7 +53,7 @@
 application lwhists_comphistfiles "-s=tests/comphistfiles *.cxx"
 private
 
-macro_append LWHists_cppflags " -fno-tree-vectorize " 
+macro_append LWHists_cppflags " -fPIC -fno-tree-vectorize " 
 
 macro_append lwhists_validate_dependencies "$(LWHists_library_dependencies)"
 macro_append lwhists_benchmark_dependencies "$(LWHists_library_dependencies)"
