--- requirements	2014-11-11 18:32:59.184147183 +0200
+++ requirements1	2014-11-12 18:12:36.705099257 +0200
@@ -21,7 +21,8 @@
 
 end_private
 
-macro_append AtlasGdb_linkopts "" \
+macro_append AtlasGdb_linkopts "$(AtlasGdb_root)/$(tag)/pkg-build-install-gdb/lib/libbfd.a \
+   $(AtlasGdb_root)/$(tag)/pkg-build-install-gdb/lib/libiberty.a -lz" \
   slc5&64 \
  " $(AtlasGdb_root)/$(tag)/pkg-build-install-gdb/lib/libbfd.a \
    $(AtlasGdb_root)/$(tag)/pkg-build-install-gdb/lib64/libiberty.a -lz " \
