--- FPEAuditor.cxx	2014-11-11 18:06:49.794255623 +0200
+++ FPEAuditor1.cxx	2014-11-14 12:33:05.054175163 +0200
@@ -118,7 +118,7 @@
   if ( m_NstacktracesOnFPE )
     {
       ATH_MSG_INFO ("Installed Signalhandler !");
-      FPEAudit::unmask_fpe();
+      //FPEAudit::unmask_fpe();
       m_SigHandInstalled=true;
     }
   FPEAudit::s_array_O[0]=NULL;
@@ -133,7 +133,7 @@
   
   feclearexcept(FE_ALL_EXCEPT);
   fesetenv (&m_env);
-  FPEAudit::mask_fpe();
+  //FPEAudit::mask_fpe();
   
   // feenableexcept (0);
   // fedisableexcept (FE_ALL_EXCEPT);
@@ -334,6 +334,6 @@
   if (!m_fpe_stack.empty()) {
     m_fpe_stack.back().second |= raised;
   }
-  if ( m_SigHandInstalled )
-    FPEAudit::unmask_fpe();
+  //if ( m_SigHandInstalled )
+    //FPEAudit::unmask_fpe();
 }
