--- requirements	2015-01-20 11:47:47.527136890 +0200
+++ requirements1	2015-01-22 08:53:08.126299317 +0200
@@ -104,10 +104,10 @@
 # ---------------------------------------------------------
 # Java stuff
 # ---------------------------------------------------------
-set JAVA_SUBDIR "sw/lcg/external/Java" \
+set JAVA_SUBDIR "/usr/lib/jvm/$(JAVAVERS)" \
 	BNL         "cernsw/java"
      
-set JAVA_BASE "${SITEROOT}/${JAVA_SUBDIR}"
+set JAVA_BASE "${JAVA_SUBDIR}"
 
 macro JDK_VENDOR "SUN_" \
       Darwin     ""
@@ -117,8 +117,8 @@
 macro JDK_FULL_VERSION "1.5.0_11" \
       Darwin           "1.5.0"
 
-set JDK_HOME  "${JAVA_BASE}/JDK/${JDK_VENDOR}${JDK_FULL_VERSION}/${JDK_ARCH}"
-set JAVA_HOME "${JDK_HOME}" target-mac106 ""
+set JDK_HOME  "${JAVA_SUBDIR}"
+set JAVA_HOME "${JDK_HOME}"
 
 set _JAVA_OPTIONS "-Xms64m -Xmx128m"
 
@@ -135,7 +135,7 @@
 macro JavaSDK_platform "" PACK "${JDK_ARCH}"
 
 macro ANT_VERSION "1.7.0"
-set   ANT_HOME    "${JAVA_BASE}/Ant/${ANT_VERSION}/share"
+set   ANT_HOME    "/usr/share/ant"
 
 path_remove PATH "Ant"
 path_prepend PATH "${ANT_HOME}/bin" noJava ""
