--- DataModelCompatSvc.cxx	2014-11-17 16:16:13.519858304 +0200
+++ DataModelCompatSvc1.cxx	2014-11-17 16:18:19.614037858 +0200
@@ -14,7 +14,7 @@
 #include "DataModelAthenaPool/CLHEPConverters.h"
 #include "RootConversions/VectorConverters.h"
 #include "RootConversions/TConvertingStreamerInfo.h"
-#include "RootConversions/TConvertingBranchElement.h"
+//#include "RootConversions/TConvertingBranchElement.h"
 #include "AthenaKernel/errorcheck.h"
 #include "GaudiKernel/MsgStream.h"
 #include "GaudiKernel/IIncidentSvc.h"
@@ -58,7 +58,7 @@
 
   // This, however, we need to do before opening any files.
   TConvertingStreamerInfo::Initialize();
-  TConvertingBranchElement::Initialize();
+  //TConvertingBranchElement::Initialize();
   DataModelAthenaPool::CLHEPConverters::initialize();
   RootConversions::VectorConverters::initialize();
   return StatusCode::SUCCESS;
