--- TConvertingBranchElement.cxx	2015-02-25 11:12:25.129855555 +0200
+++ TConvertingBranchElement.cxx1	2015-02-25 11:13:29.734459482 +0200
@@ -1107,7 +1107,7 @@
    // ReadSequence doesn't work here, since it gets structure offsets
    // from TConfiguration, and those haven't been adjusted to take
    // into account the use of the temporary conversion objects.
-#if ROOT_VERSION_CODE >= ROOT_VERSION(6,1,0) || (ROOT_VERSION_CODE>=ROOT_VERSION(5,34,22) && ROOT_VERSION_CODE<ROOT_VERSION(6,0,0))
+#if ROOT_VERSION_CODE >= ROOT_VERSION(6,1,0) || (ROOT_VERSION_CODE>=ROOT_VERSION(5,34,21) && ROOT_VERSION_CODE<ROOT_VERSION(6,0,0))
    // FIXME!
    std::abort();
 #else
@@ -1321,7 +1321,7 @@
   TClass* cl_orig = 0;
   if (fConvClass && fID > -1) {
     TStreamerInfo* si = GetInfo();
-#if ROOT_VERSION_CODE >= ROOT_VERSION(6,1,0) || (ROOT_VERSION_CODE>=ROOT_VERSION(5,34,22) && ROOT_VERSION_CODE<ROOT_VERSION(6,0,0))
+#if ROOT_VERSION_CODE >= ROOT_VERSION(6,1,0) || (ROOT_VERSION_CODE>=ROOT_VERSION(5,34,21) && ROOT_VERSION_CODE<ROOT_VERSION(6,0,0))
     branchElem = si->GetElem(fID);
     if (branchElem) {
 #else
