--- requirements	2014-11-11 18:19:48.379257687 +0200
+++ requirements1	2014-11-13 15:28:02.384332796 +0200
@@ -67,9 +67,9 @@
 # is a prefix of the name of another causes problems for parallel builds.
 # So we renamed RootConversionsTestX to TestRootConversionsX.
 
-apply_pattern lcgdict dict=TestRootConversions1 selectionfile=../test/selection_test1.xml headerfiles="../test/RootConversionsTest1Dict.h"
-macro_append reflex_dictTestRootConversions1_options " -I. "
+#apply_pattern lcgdict dict=TestRootConversions1 selectionfile=../test/selection_test1.xml headerfiles="../test/RootConversionsTest1Dict.h"
+#macro_append reflex_dictTestRootConversions1_options " -I. "
 
-apply_pattern lcgdict dict=TestRootConversions2 selectionfile=../test/selection_test2.xml headerfiles="../test/RootConversionsTest2Dict.h"
-macro_append reflex_dictTestRootConversions2_options " -I. "
+#apply_pattern lcgdict dict=TestRootConversions2 selectionfile=../test/selection_test2.xml headerfiles="../test/RootConversionsTest2Dict.h"
+#macro_append reflex_dictTestRootConversions2_options " -I. "
 
