--- requirements	2015-05-06 15:43:03.564822000 +0000
+++ requirements1	2015-05-06 17:58:06.554822000 +0000
@@ -352,6 +352,14 @@
 tag slc615   host slc6
 tag slc616   host slc6
 
+
+# ------------------ ARM 64 ------------
+tag aarch64-ubuntu14-gcc49-opt 	Linux ubuntu14  64  gcc-4.9  opt    aarch64-ubuntu14-gcc49
+tag aarch64-ubuntu14-gcc49-opt       Linux ubuntu14  64  gcc-4.9  debug  aarch64-ubuntu14-gcc49
+
+tag ubuntu14 host-ubuntu
+tag ubuntu	host-ubuntu
+
 # ------------------ Special Platform tags ------------
 
 tag CERNDISTCC CERN distcc
