--- xml.cxx	2014-11-12 19:08:27.279057553 +0200
+++ xml1.cxx	2014-11-12 19:17:26.085659607 +0200
@@ -103,7 +103,7 @@
 
 /*--------------------------------------------------------------------------*/
 /* Data structures for the actual XML dump.  */
-
+int plugin_is_GPL_compatible;
 /* A "dump node" corresponding to a particular tree node.  */
 typedef struct xml_dump_node
 {
@@ -362,22 +362,22 @@
     xml_add_node(xdi, node, complete))
 #endif
 /* Get the revision number of this source file.  */
-const char* xml_get_xml_c_version()
-{
-  const char* revision = GCC_XML_C_VERSION;
-  char* version = (char*)xmalloc(strlen(revision)+1);
-  const char* in=revision;
-  char* out = version;
-  while(*in && *in != ':') { ++in; }
-  if(*in) { ++in; }
-  while(*in && *in == ' ') { ++in; }
-  while(*in && *in != ' ' && *in != '$')
-    {
-    *out++ = *in++;
-    }
-  *out = 0;
-  return version;
-}
+//const char* xml_get_xml_c_version()
+//{
+  //const char* revision = GCC_XML_C_VERSION;
+ // char* version = (char*)xmalloc(strlen(revision)+1);
+ // const char* in=revision;
+  //char* out = version;
+ // while(*in && *in != ':') { ++in; }
+ // if(*in) { ++in; }
+ // while(*in && *in == ' ') { ++in; }
+ // while(*in && *in != ' ' && *in != '$')
+  //  {
+  //  *out++ = *in++;
+ //   }
+///  *out = 0;
+//  return version;
+//}
 
 /* Main XML output function.  Called by parser at the end of a translation
    unit.  Walk the entire translation unit starting at the global
@@ -445,7 +445,7 @@
 #if defined(GCCXML_PLUGIN_VERSION_FULL)
   fprintf (file, " version=\"" GCCXML_PLUGIN_VERSION_FULL "\"");
 #endif
-  fprintf (file, " cvs_revision=\"%s\"", xml_get_xml_c_version());
+  //fprintf (file, " cvs_revision=\"%s\"", xml_get_xml_c_version());
   fprintf (file, ">\n");
 
   /* Dump the complete nodes.  */
